docs/code-standards.md

```diff
@@ -0,0 +1,39 @@
+# Code & Documentation Standards
+
+This document details our goals for code quality, coverage and documentation standards. At the time
+of this writing, these are standards that all of our code may not meet. We should always try to 
+improve our codebase and documentation to meet them.
+
+While we do not currently aim to adhere completely to the 
```

Comment:
Because that's large document and we haven't discussed in full yet. This document may change once we do.